src/pageObject.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+
+    this.slug = slug;
+    this.url += this.slug;
```

Comment:
U can add a slug instead of creating a a new pair key: value ` this.url = url + slug;`

Reply:
Ok, thanks. I get it)